src/Types/JsonWsp.purs

```diff
@@ -218,9 +246,9 @@ parseValue :: Object Json -> Either JsonDecodeError Value
 parseValue outer = do
   o <- getField outer "value"
   coins <- parseFieldToBigInt o "coins" <|> (parseFieldToInt o "coins") <|> Left (TypeMismatch "Expected 'coins' to be an Int or a BigInt")
-  (assetsJson :: {}) <- getField o "assets"
-  -- note 'coins' is being sent as a number, in some cases this may exceed the max safe 
-  -- representation of a Number, we may need to parse this up from a string instead of from 
+  (_assetsJson :: {}) <- getField o "assets"
```

Comment:
so we need a ticket to support native assets right?   let's prefer tickets over TODO's